day1.md

```diff
@@ -0,0 +1,119 @@
+## Day1 Mon. 09/05/16 [Tokyo 東京]
+---
+
+06:30         [Landed @NRT Airport](#landed)<br />
+09:00         [Tokyo Station](#tokyo_station)<br />
+10:00 - 12:00 [Food & Post Office](#first_meal)<br />
+12:30 - 13:40 [Ikebukuro Station & Airbnb place](#ikebukuro_station)<br />
+15:40 - 20:30 [Odaiba Onsen](#odaiba_onsen)
+
+---
+After we <a name="landed">landed</a> around 06:30 am, we picked up our JR passes<sup>[1](#jrpass)</sup> and pocket Wifi<sup>[2](#pocketWifi)</sup> and got some cash<sup>[3](#cash)</sup> from an ATM at the Narita International Airport. Then we took the Narita Express, covered by our JR Passes, to get to Tokyo!
```

Comment:
"Then we took the Narita Express, covered by our JR Passes, to get to Tokyo!"
-> remove the comma before "covered"